MAUI/Masked-Entry/Overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: About .NET MAUI MaskedEntry control | Syncfusion
+description: Learn all about the introduction of Syncfusion MAUI MaskedEntry (SfMaskedEntry) control with essential features and more here.
```

Comment:
Description: Learn about the introduction of Syncfusion MAUI MaskedEntry (SfMaskedEntry) control with essential features and more.